src/components/Requirements/SubRequirement.vue

```diff
@@ -218,6 +218,18 @@ export default defineComponent({
     color: { type: String, required: true },
     tourStep: { type: Number, required: true },
   },
+  emits: {
+    'modal-open': (open: boolean) => typeof open === 'boolean',
+    changeToggleableRequirementChoice(id: string, option: string) {
```

Comment:
why do we have `'modal-open'` as a string and `changeToggleableRequirementChoice(...)`?

Reply:
They have the same runtime effect, the only difference is that `'modal-open'` without quote is not a legal JS identifier.